Source/Cotter.swift

```diff
@@ -121,6 +121,7 @@ public class Cotter {
         // hide the close button (Optional)
         self.pinVC.hideCloseButton = hideClose
         
+        Config.instance.parent = vc
```

Comment:
this is not the fix. in fact I think I think you should remove the `parent` variable from `Config.instance`.

The fix should be done in the `actionHandler` function for the `AlertServiceDelegate` protocol inside `PINViewController.swift`. Just remove the line about popping the view controller

Reply:
oh ya i forgot that its just supposed to skip